styles.css

```diff
@@ -1,786 +1,786 @@
+  color: #dfe1e6;
+}
+
+.sep2 {
+  margin-bottom: 12%;
+  width: 95%;
+  color: #dfe1e6;
+  padding-bottom: 35%;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.r-a {
+  padding-right: 5%;
+  padding-bottom: 5%;
+}
+
+@media screen and (min-width: 768px) {
+  .container {
+    margin: auto 10%;
+  }
+
+  .contact-me {
+    background-image: url('./images/contact-original.png');
+    background-repeat: no-repeat;
+    background-size: 90%;
+    background-position: 90px 90px !important;
+  }
+
+  .left-img {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 6%;
+    padding: 2.8rem;
+    gap: 6%;
+  }
+
+  .link {
+    padding: 0;
+  }
+
+  .right-img {
+    display: flex;
+    flex-direction: row-reverse;
+    column-gap: 6%;
+    padding: 2.8rem;
+    gap: 6%;
+  }
+
+  .first-image {
+    border-radius: 0.8rem;
+    width: 100%;
+  }
+
+  .sep {
+    color: #dfe1e6;
+  }
+
+  .cont-btn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #6070ff;
+    background-color: #fff;
+    font-size: 1.7rem;
+    font-weight: 500;
+    padding: 1.2rem 1.6rem;
+    border: none;
+    border-radius: 0.8rem;
+    cursor: pointer;
+    width: inherit;
+    margin: 0 auto;
+    margin-top: 2%;
+  }
+
+  .pp-1 {
+    display: none;
+  }
+
+  .frontend {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .card-details {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 2rem;
+  }
+
+  .space-desk {
+    display: none;
+  }
+
+  /* NAVBAR SECTION  */
+  .navbar {
+    padding: 2.3rem 6.8rem;
+  }
+
+  .dashboard {
+    display: none;
+  }
+
+  .end-1 {
+    width: 100%;
+  }
+
+  .text-1 {
+    color: #344563;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    padding-right: 10%;
+    padding-bottom: 3%;
+  }
+
+  .nav-btn {
+    display: none;
+  }
+
+  ul.list-nav {
+    list-style: none;
+    display: flex;
+    gap: 3.2rem;
+  }
+
+  .list-nav a:link,
+  .list-nav a:visited {
+    text-decoration: none;
+    color: #344563;
+    display: flex;
+    align-items: center;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .special {
+    padding-top: 21%;
+    width: inherit;
+    border-top-right-radius: 55pt;
+  }
+
+  .list-nav a:hover,
+  .list-nav a:active {
+    border-bottom: 0.3rem solid #6070ff;
+  }
+
+  .connect-1 {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .about-p {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  /* HEADLINE PART */
+  .container-all {
+    opacity: 3;
+    width: 100%;
+    border-radius: 0;
+    padding-top: 23%;
+    gap: 2.2rem;
+    padding-bottom: 12%;
+    border-bottom-left-radius: 75pt;
+  }

```

Comment:
- [ ] Maybe while trying to implement the required changes from the previous reviewer, you mistakenly remove the background image of your headline section. Kindly return it as to match the figma design.

### Your Design
![your-headline-section-desktop](https://github.com/hetrox8/portfolio1/assets/26597432/924fb7a1-f2cb-4def-8e37-df2ac17cee7a)

### Figma Design
![figma-headline-section-desktop](https://github.com/hetrox8/portfolio1/assets/26597432/ca520ac1-f910-486d-84eb-5284c6a39bb8)